src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+}
+
+.nav__list {
+  display: flex;
```

Comment:
Good job on using a class selector for styling the list, instead of the 'ul' tag. This is a good practice as it avoids potential conflicts with other 'ul' tags on the page.